go-controller/pkg/ovn/master.go

```diff
@@ -522,8 +522,7 @@ func (oc *DefaultNetworkController) syncNodesPeriodic() {
 	}
 }
 
-// We only deal with cleaning up nodes that shouldn't exist here, since
-// watchNodes() will be called for all existing nodes at startup anyway.
+// We only deal with cleaning up nodes that shouldn't exist here.
```

Comment:
This comment holds true as it was. Curious on why did you think about changing it?
https://github.com/ovn-org/ovn-kubernetes/blob/0bfb6bd72363eb33c1d2e76e9d20a3402e7b3a90/go-controller/pkg/ovn/base_network_controller.go#L679

Reply:
removed - the comment specifies `watchNodes()` which I couldn't find but `WatchNodes()` does exist